integration/integration_test.go

```diff
@@ -485,7 +485,7 @@ func testAuditOn(t *testing.T, suite *integrationTestSuite) {
 			start := findByType(events.SessionStartEvent)
 			require.Equal(t, first, start)
 			require.Equal(t, 0, start.GetInt("bytes"))
-			require.NotEmpty(t, start.GetString(events.SessionEventID))
+			require.Equal(t, string(sessionID), start.GetString(events.SessionEventID))
```

Comment:
nit: _not really related to this fix_, but it's strange that this test creates a `sessionID` variable from `session.ID` and then uses a mix of both throughout.

Reply:
Yeah, at some point we should use the `session.ID` type everywhere. There's still a fair amount of code that expects a string instead, which results in a bunch of conversions back and forth.